widget_examples/multi_file_viewer/widgets.json

```
@@ -12,14 +12,16 @@
       {
         "type": "endpoint",
         "paramName": "whitepaper",
-        "value": "http://localhost:5011/whitepapers",
+        "value": ["Bitcoin"],
         "label": "Whitepaper",
         "description": "Whitepaper to display.",
         "optionsEndpoint": "/whitepapers",
         "show": false,
         "optionsParams": {
           "category": "$category"
-        }
+        },
+        "multiSelect": true,
+        "role": ["fileSwitcher"]
```

Nit:
```suggestion
        "role": ["fileSelector"]
```

(I don't mind it being `fileSwitcher` either)